bootcfg/rpc/select.go

```diff
@@ -0,0 +1,30 @@
+package rpc
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/coreos/coreos-baremetal/bootcfg/server"
+	pb "github.com/coreos/coreos-baremetal/bootcfg/server/serverpb"
+)
+
+// selectServer wraps a bootcfg Server to be suitable for gRPC registration.
+type selectServer struct {
+	// bootcfg Server
+	srv server.Server
+}
+
+func newSelectServer(s server.Server) pb.SelectServer {
+	return &selectServer{
+		srv: s,
+	}
+}
+
+func (s *selectServer) SelectGroup(ctx context.Context, req *pb.SelectGroupRequest) (*pb.SelectGroupResponse, error) {
+	group, err := s.srv.SelectGroup(ctx, req)
+	return &pb.SelectGroupResponse{Group: group}, err
```

Comment:
Maybe in a separate PR...
it's best to use standard gRPC error codes with your returned errors.

https://godoc.org/google.golang.org/grpc#Errorf